<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connected</class>
 <widget class="QMainWindow" name="Connected">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chatroom</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Chatroom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connected Clients</string>
    </property>
   </widget>
   <widget class="QListWidget" name="connectedclient_list">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="chatroom_list">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="Chatroom_label2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chat rooms (Group chat)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onetoone_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;</string>
    </property>
    <property name="text">
     <string>1 : 1 Chat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createroom_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinroom_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;</string>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeconnected_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcome_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome, &lt;User&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="connected_refresh_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>0</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="connected_refresh_group_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
